CHANGELOG.md

```diff
@@ -1,19 +1,22 @@
 # CHANGELOG
 
 ## Next (upcoming release)
+TBD
```

Comment:
Maybe we should leave this section off for release branches?

Reply:
Ok, will do! This is against main but I'll port this over to release and remove the TDB/next section